cfme/tests/networks/test_sdn_balancers.py

```diff
@@ -45,3 +45,21 @@ def test_sdn_balancers_detail(provider, network_prov_with_load_balancers):
         for balancer in prov.balancers.all():
             check = balancer.network_provider
             assert check is not None
+
+
+# only one provider is needed for that test, used Azure as it has balancers
+@pytest.mark.provider([AzureProvider], scope='module', override=True)
+@pytest.mark.parametrize('visibility', [True, False], ids=['visible', 'notVisible'])
+def test_sdn_balancers_tagvis(check_item_visibility, visibility, network_prov_with_load_balancers):
+    """ Tests network provider and its items honors tag visibility
+    Prerequisites:
```

Comment:
Will the automation take care of any of these prerequisites? If not, can we check for some of these prerequisites in the test and throw Exceptions that tell us a prerequisite has not been met, so that the reason for a test failing is clear?

Reply:
All these prerequisites are done with the fixtures passed to test, so if any fails we will see it in the run